File: mtool/reference/legacy-mtool-templates/dbclasses_template_system_default/objectivecm/template-dataclass-list-serialize.m
- (NSArray<NSDictionary*>*)Serialize
{
    NSArray* result = [[NSArray<NSDictionary*> alloc] init];
    for (int i = 0; i < self.count; i++) {
        NSDictionary* item = [self[i] Serialize];
        [result arrayByAddingObject:item];
    }
    return result;
}

File: mtool/reference/legacy-mtool-templates/dbclasses_template_system_default/swift/template-dataclass-list-serialize.swift
    func Serialize() -> Array<Dictionary<String, Any>>
    {
        var result:Array<Dictionary<String, Any>> = Array<Dictionary<String, Any>>();
        for i in 0 ..< _backendArray.count {
            let item:Dictionary<String, Any> = _backendArray[i].Serialize();
            result.append(item);
        }
        // == START OF EDITABLE AREA FOR SERIALIZE ==
        // == END OF EDITABLE AREA FOR SERIALIZE ==
        return result;
    }
